src/PrestaShopBundle/Resources/views/Admin/Improve/Shipping/Carriers/Blocks/showcase_card.html.twig

```diff
@@ -0,0 +1,50 @@
+{#**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *#}
+
+{% trans_default_domain 'Admin.Shipping.Feature' %}
+
+{% if not isShowcaseCardClosed %}
+  <div id="carriersShowcaseCard" class="helper-card card">
+    <div class="helper-card__left helper-card__carrier shape-one">
+      <img src="{{ asset('themes/new-theme/public/helper-card/carrier@3x.png') }}">
+    </div>
+    <div class="helper-card__right">
+      <h2>{{ 'Manage your carriers'|trans }}</h2>
+      <p>{{ 'Create carriers to let your customers choose their best shipping option in terms of delivery dates or fees. Start with editing/deleting the default "My carrier", and add as many options as you need.'|trans }}</p>
+      <a class="btn btn-outline-secondary"
+         href="{{ documentation_link('carrier') }}"
+         target="_blank"
+      >
+        {{ 'Learn more'|trans({}, 'Admin.Actions') }}
+      </a>
+    </div>
```

Comment:
@Julievrz, you can check the wording with this document that lists all the wording related to the empty states or helper cards: https://docs.google.com/document/d/1e_yu6f3Eq1c430k-a2QA9WLZwbwRSPsIeM_jhp09Jdg/edit. ;-)

Reply:
@LouiseBonnard I checked in the document and the wording seems to match. However, I don't understand what you mean by "empty states" and I couldn't find the wording on the software with the machine shuffle...

Reply:
Perfect then, I resolve this conversation.